Doc/howto/free-threading-python.rst

```diff
@@ -0,0 +1,119 @@
+.. _freethreading-python-howto:
+
+*********************************
+Python Support for Free Threading
+*********************************
+
+Starting with the 3.13 release, CPython has experimental support for running
+with the :term:`global interpreter lock` (GIL) disabled in a configuration
+called :term:`free threading`.  This document describes the implications of
+free threading for Python code.  See :ref:`freethreading-extensions-howto` for
+information on how to write C extensions that support the free-threaded build.
+
+
+Installation
+============
+
+Starting with Python 3.13.0b2, the offical macOS and Windows installers
```

Comment:
3.13.0 final is the next release: maybe we can gloss over the actual pre-release it was added? Plus a typo:

```suggestion
Starting with Python 3.13, the official macOS and Windows installers
```